.gitignore

```diff
@@ -151,4 +151,12 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-
+# unit test files/directories
+tests/data/geo_cslc_s1.yaml
+tests/data/2022-10-16_0000_Rosamond-corner-reflectors.csv
+tests/data/S1A_IW_SLC__1SDV_20221016T015043_20221016T015111_045461_056FC0_6681.zip
+tests/data/orbits
+tests/data/test_dem.tiff
+tests/data/test_burst_map.sqlite3
+tests/product
+tests/scratch
```

Comment:
If these are getting created in the test directory and we want to ignore them, what if we use a temporary pytest directory that'll get removed?

Reply:
E.g. in conftest.py, using https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture

```python

@pytest.fixture(scope="session")
def geocode_slc_params(tmp_path_factory):

...
    # get test working directory
    test_path = pathlib.Path(__file__).parent.resolve()
    output_path = tmp_path_factory.mktemp("output")

...
      cfg = f_template.read().replace( '@TEST_PATH@', str(output_path)).\

```


But now that i've written this out, I'm thinking you may have stored it in the `tests/` directory so that you could inspect the output, so feel free to ignore this suggestion if that's the case